models/src/main/protobuf/DeployServiceCommon.proto

```diff
@@ -49,6 +49,12 @@ message DataAtNameQuery {
   Par name = 2;
 }
 
+message DataAtParQuery {
```

Comment:
Please rename it to `DataAtNameByBlockQuery`.

Reply:
Resolved in [commit](https://github.com/rchain/rchain/pull/3592/commits/dcbc0f0f25a475c8ea7503d55ceca8e3b177177d)